docs/core/resilience/snippets/http-resilience/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Net;
+using Microsoft.Extensions.DependencyInjection;
+using Polly;
+using Polly.CircuitBreaker;
+using Polly.Retry;
+
+var services = new ServiceCollection();
+
+var builder = services.AddHttpClient<ExampleClient>(
+    configureClient: static client =>
+    {
+        client.BaseAddress = new("https://jsonplaceholder.typicode.com");
+    });
+
+builder.AddStandardResilienceHandler(options =>
+{
+    options.RetryOptions.BackoffType = DelayBackoffType.Exponential;
```

Comment:
`Exponential` is already a default on these options.

Reply:
According to Visual Studio, it's default is `Constant`. I think I see why that is though:

![image](https://github.com/dotnet/docs/assets/7679720/02d67678-a90d-41fb-8587-57d8324f7b3d)

Reply:
Yeah, the HttpRetryStrategyOptions change the default in the constructor. 

Dunno if there is a good solution to this.